api/models/fda_application.js

```diff
@@ -30,6 +30,9 @@ const FDAApplication = BaseModel.extend({
     url: function () {
       return helpers.urlFor(this);
     },
+    type: function () {
+      return this.id.match(/[A-Z]+/gi)[0];
```

Comment:
The IDs use the format `XXX000000` (e.g. `NDA00000`, `ANDA00000`), so this will return the first group of non-numeric values there.

You're right to be concerned, though. I added a new check in case the id doesn't start with letters, just to be safer :+1: